taesch/lib/view/screen/settings_screen.dart

```diff
@@ -23,20 +23,121 @@ class _SettingsScreenState extends State<SettingsScreen> {
     logger.log(
         level: LogLevel.info,
         logMessage: LogMessage(message: "entered settings screen"));
-    return SwitchListTile(
-      title: Text(widget._vm.switchTitle),
-      value: ThemeController.of(context).darkTheme,
-      onChanged: (bool value) {
-        logger.log(
-            level: LogLevel.info,
-            logMessage: LogMessage(
-                message:
-                    "switched ${widget._vm.switchTitle} button to $value"));
-        setState(() {
-          ThemeController.of(context).darkTheme = value;
-        });
-      },
-      secondary: const Icon(Icons.sunny),
+    return Scaffold(
+      body: Column(
+        children: [
+
+          // ursprünglich war es nur "return SwitchListTile(...)"
```

Comment:
Kommentar noch gebraucht (vorherige Stände lassen sich in Commits ablesen, die vorher gemacht wurden)